Credify/Credify/WebView/WebPresenter.swift

```diff
@@ -251,10 +261,10 @@ class WebPresenter: WebPresenterProtocol {
         }
         
         switch context {
-        case .offer(_, _):
-            // Both buttons are hidden in the Offer Pending page
-            if url.starts(with: "\(Constants.WEB_URL)/pending-offer") {
-                return false
+        case .mypage(_), .serviceInstance(_, _, _):
+            // In this case, the back button is visible
+            if url.starts(with: pendingUrl) || url.starts(with: canceledUrl) {
```

Comment:
Please move this logic to one single place as helper function. You have the same lines at several places.

Reply:
I updated my code